mockgcp/mocksecretmanager/secretversions.go

```diff
@@ -286,9 +292,12 @@ func (s *SecretsV1) DestroySecretVersion(ctx context.Context, req *pb.DestroySec
 		return nil, err
 	}
 
+	now := timestamppb.Now()
```

Comment:
Insert standard diatribe against using actual time in tests.

Reply:
True, we have to deal with it in normalization.  We do want updateTime to be not-a-constant, because we might want to use it for change detection though...